stream-chat-android-ui-components/src/main/res/layout/stream_ui_message_threads_footnote.xml

```diff
@@ -1,39 +1,108 @@
+        android:layout_height="@dimen/stream_ui_message_thread_reply_avatar_size"
+        app:layout_constraintStart_toEndOf="@id/threadsOrnamentLeft"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:streamUiAvatarBorderWidth="0dp"
+        app:streamUiAvatarOnlineIndicatorEnabled="false"
+        tools:src="@tools:sample/avatars"
+        tools:visibility="gone"
+        />
+    
+    <io.getstream.chat.android.ui.avatar.AvatarView
+        android:id="@+id/firstTheirUserImage"
+        android:layout_width="@dimen/stream_ui_message_thread_reply_avatar_size"
+        android:layout_height="@dimen/stream_ui_message_thread_reply_avatar_size"
+        android:layout_marginStart="@dimen/stream_ui_message_thread_reply_avatar_half_size"
+        app:layout_constraintStart_toStartOf="@id/secondTheirUserImage"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_goneMarginStart="0dp"
+        app:streamUiAvatarBorderWidth="0dp"
+        app:streamUiAvatarOnlineIndicatorEnabled="false"
+        tools:src="@tools:sample/avatars"
+        tools:visibility="gone"
         />
 
     <TextView
         android:id="@+id/threadRepliesButton"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginLeft="@dimen/stream_ui_spacing_tiny"
-        android:layout_marginRight="@dimen/stream_ui_spacing_tiny"
+        android:layout_marginStart="@dimen/stream_ui_spacing_small"
+        android:layout_marginEnd="@dimen/stream_ui_spacing_small"
         android:textColor="@color/stream_ui_accent_blue"
         android:textAppearance="@style/StreamUiTextAppearance.FootnoteBold"
+        app:layout_constraintStart_toEndOf="@id/firstTheirUserImage"
+        app:layout_constraintEnd_toStartOf="@id/firstMineUserImage"
+        app:layout_constraintTop_toTopOf="@id/firstTheirUserImage"
         tools:text="100 Thread Replies"
         />
 
+    <io.getstream.chat.android.ui.avatar.AvatarView
+        android:id="@+id/firstMineUserImage"
+        android:layout_width="@dimen/stream_ui_message_thread_reply_avatar_size"
+        android:layout_height="@dimen/stream_ui_message_thread_reply_avatar_size"
+        android:layout_marginEnd="@dimen/stream_ui_message_thread_reply_avatar_half_size"
+        app:layout_constraintEnd_toEndOf="@id/secondMineUserImage"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_goneMarginStart="0dp"
+        app:streamUiAvatarBorderWidth="0dp"
+        app:streamUiAvatarOnlineIndicatorEnabled="false"
+        tools:src="@tools:sample/avatars"
+        />
+
+    <io.getstream.chat.android.ui.avatar.AvatarView
+        android:id="@+id/secondMineUserImage"
+        android:layout_width="@dimen/stream_ui_message_thread_reply_avatar_size"
+        android:layout_height="@dimen/stream_ui_message_thread_reply_avatar_size"
+        android:layout_marginEnd="@dimen/stream_ui_message_thread_reply_avatar_half_size"
```

Comment:
This margin seems unnecessary, it results in the avatars being disconnected from the ornament, like this:

![image](https://user-images.githubusercontent.com/12054216/104822035-3f6a1480-5840-11eb-8d49-54f92ad8e085.png)

vs the behaviour on the other side:

![image](https://user-images.githubusercontent.com/12054216/104822052-5577d500-5840-11eb-9b12-e1095cbf9641.png)